packages/viewer/src/components/viewer-hit-result-indicator/lib/point.ts

```diff
@@ -0,0 +1,79 @@
+import { Angle, Matrix4, Quaternion, Ray, Vector3 } from '@vertexvis/geometry';
+
+import { MeshPoints } from '../../../lib/transforms/mesh';
+import { FrameCameraBase } from '../../../lib/types';
+
+export function computePointNdcValues(
+  transform: Matrix4.Matrix4,
+  camera: FrameCameraBase,
+  direction: Vector3.Vector3,
+  rectangleSize: number
+): MeshPoints {
+  const transformedDirection = Vector3.transformMatrix(
+    direction,
+    Matrix4.makeRotation(Quaternion.fromMatrixRotation(transform))
+  );
+
+  const position = Vector3.fromMatrixPosition(transform);
+
+  const worldX = Vector3.normalize(
+    Vector3.cross(transformedDirection, Vector3.normalize(camera.viewVector))
+  );
+  const worldY = Vector3.normalize(Vector3.cross(transformedDirection, worldX));
+  const xRay = Ray.create({
+    origin: position,
+    direction: worldX,
+  });
+  const yRay = Ray.create({
+    origin: position,
+    direction: worldY,
+  });
+
+  const bottomLeft = Ray.at(xRay, -(rectangleSize / 2.5));
```

Comment:
What is significant here about 2.5?

Reply:
Nothing specific, just an arbitrary value similar to what we do for the arrow/axis point computation. That being said, we could probably at least pull this into a constant in the `indicator.ts` file when we pass this value and remove the constant here

Reply:
Good idea - let's do that. Especially if it makes sense for all of these to be consistent.